tools/install-source/XamarinSourcesPathMangler.cs

```diff
@@ -84,21 +84,7 @@ static bool IsRunrime (string path)
 		/// <param name="path">Path.</param>
 		string GetSourcePathForGeneratedPath (string path)
```

Comment:
One liner because I love you.

`string GetSourcePathForGeneratedPath (string path) => path;`